CMakeLists.txt

```diff
@@ -566,7 +567,7 @@ option(WITH_SYSTEM_BOOST "require and build with system Boost" OFF)
 # Boost::thread depends on Boost::atomic, so list it explicitly.
 set(BOOST_COMPONENTS
   atomic chrono thread system regex random program_options date_time
-  iostreams context coroutine)
+  iostreams context coroutine filesystem)
```

Comment:
i don't understand why we need Boost.filesystem? could you shed some light on this?

Reply:
this is required by `boost::process` - I use it to execute `luarocks`